### ISSUE TITLE
can login with same character multiple times.

### ISSUE THREAD

**Issue by zhyrohaad** [zhyrohaad/AlasiyaEvE#53]
*see title*
![Screenshot (14)](https://user-images.githubusercontent.com/6365054/64826494-79f5da00-d586-11e9-87d8-c84a52ca5904.png)

this is full screenshot from dual display.
left side is first login, and has chat windows.
right side is second login.  there was an error msg about "you are already in the channel...." but i didnt get it in time.

**Comment by ryllae** [comment]
I've never been able to duplicate this, I always get an error from the login screen.
![Capture](https://user-images.githubusercontent.com/51723245/70019089-c7626180-154d-11ea-8940-2b600ce2547e.PNG)

Using the AlasiyaLauncher, I get the same error, then the client never updates to let me try again.
![Annotation 2019-12-02 215158](https://user-images.githubusercontent.com/51723245/70019156-f973c380-154d-11ea-84a2-d3e5b28405ac.png)

**Comment by THUNDERGROOVE** [comment]
The second bit of your reply is on me.  I haven't gotten around to fixing
the client hook yet.

**Comment by zhyrohaad** [comment]
woot!!!  i can post an error for someone else to fix!!?!?
sweet dude!!
LOL

**Comment by zhyrohaad** [comment]
after a db query update, i cannot reproduce this error.
im gonna call it good.....except for groove's part ;)
